docs/pages/workflow/debugging.md

```diff
@@ -55,41 +56,65 @@ This might indicate that there is a performance issue. You likely need to run yo
+
+The Developer Menu provides multiple options. A few are pretty self-explanatory, like:
+
+- Reload: reload your app - usually not necessary if you have Fast Refresh enabled
+- Copy link to clipboard: Copy the [`exp://` link to your app](../guides/linking.md)
+- Go to Home: Leave your app and navigate back to the Expo Go app homescreen
+- Enable/Disable Fast Refresh: Toggle automatic refreshing the JS bundle whenever you make changes to files in your project
 
 Now let's explore some of the more exciting functionalities...
 
+#### Debug Remote JS
+
+Opens a React Native Debugger tab in your browser, thanks to [`@react-native-community/cli-debugger-ui`](https://github.com/react-native-community/cli/tree/main/packages/cli-debugger-ui):
+
+<ImageSpotlight alt="The Expo Go Element Inspector, showing details about an element after inspecting it" src="/static/images/react-native-community-cli-debugger-ui.png" containerStyle={{ paddingBottom: 0 }} />
+
+This allows you to use your browser's DevTools for the Console tab (for using `console.log`).
+
+⚠️ The Network tab will not work out of the box! To enable the Network tab and other debugging tools, additional setup is required - see the [React Native Debugger](#react-native-debugger) and [React DevTools](#debugging-with-react-devtools) sections below.
+
 #### Show Performance Monitor
 
-This opens up a small window giving you performance information of your app. You will see:
+Opens up a small window giving you performance information of your app. You will see:
 
 - RAM usage of your project
 - JavaScript heap (this is an easy way to know of any memory leaks in your application)
 - 2 numbers for Views, the top indicates the number of views for the screen, the bottom indicates the number of views in the component
 - Frames Per Second for the UI and JS threads. The UI thread is used for native Android or iOS UI rendering. The JS thread is where most of your logic will be run, including API calls, touch events, etc.
 
-#### Toggle Element Inspector & Debug Remote JS
+#### Show/Hide Element Inspector
+
+Shows the Element Inspector overlay:
+
+<ImageSpotlight alt="The Expo Go Element Inspector, showing details about an element after inspecting it" src="/static/images/expo-go-element-inspector.png" containerStyle={{ paddingBottom: 0 }} />
+
+This overlay has capabilities to:
 
-Both of these work best in tandem with either `react-native-debugger` or `react-devtools`. Read on to see more!
+1. Inspect: Inspect elements
+2. Perf: Show Performance overlay
+3. Network: Show network details
+4. Touchables: Highlight touchable elements
 
-## React Native Debugger
+### React Native Debugger
 
 The React Native Debugger includes a lot of the tools listed later in this page, all bundled into one, including React-DevTools ([guide below](#debugging-with-react-devtools)) and network request inspection. For this reason, if you use one tool on this page, it should probably be this one!
```

Comment:
```suggestion
The React Native Debugger includes many tools listed later on this page, all bundled into one, including [React DevTools](#debugging-with-react-devtools) and network request inspection. For this reason, if you use one tool from this page, it should probably be this one.
```

Unrelated to the changes in this PR but I think this is a good opportunity to update some here to match Expo's writing style guide.